EventsExpress.Db/EF/AppDbContext.cs

```diff
@@ -185,7 +197,91 @@ protected override void OnModelCreating(ModelBuilder builder)
                 .HasOne(uei => uei.Inventory)
                 .WithMany(i => i.UserEventInventories)
                 .HasForeignKey(uei => uei.InventoryId).OnDelete(DeleteBehavior.Restrict);
+        }
+
+        public void SaveTracks()
```

Comment:
Consider splitting this method into smaller ones, so it becomes more readable. Currently it spans for 70 lines, which is too big.